### DISCUSSION TITLE
Where to get an OBF file from?

### DISCUSSION THREAD

**Discussion by JulianGro** [osmandapp/OsmAnd#16401] [Maps Related]
I am trying to see if I fix some routing issues in my hometown by following https://osmand.net/docs/technical/contributions/debug-obf-routing-on-pc/
I am stuck at the step "copy the OBF file from your phone or tablet into that folder". 
For one, is there no way to just download that on my computer?

And if there isn't, how do I get "the OBF file" from my phone? Like where can I export it from, or where is it normally located?

**Comment by mariush444** [comment]
USB cable and brows your phone doesn't work?
Can you see your files on Android with TotalCommander or MiXplorer?
What Android ver. do you use?

**Reply by JulianGro** [reply]
I can browse my files, but I have no idea where those specific files are.

I am on Android 9.

**Comment by mariush444** [comment]
Can you verify: osmand settings -> data storage folder ?
usually it is in the phone's memory or in the sd card in folder Android/data/net.osmand or Android/data/net.osmand.plus

**Comment by le-stro** [comment]
Hi, 

even if the title doesn't properly fit any more the previous discussion does.

Using OsmAnd 4.4.7  , Android 13 on Redmi Note 12.
Other *.obf file downloaded through the app are located in subfolder ..... /net.osmand / files

So I created a obf map file. With the help of FV File Manager stored the file in ..../ Android / data / net.osmand
It's not possible to store  a file in a subfolder of ..../net.osmand. At least I couldn't find a way to do so. (Maybe an Android 13 issue ?)

Questions:
How to active my map?
Will OsmAnd recognize a new obf file only if it's stored in ..../files?
Or sufficient to have the obf file in ..../net.osmand?

Any help is welcome.
